melissa/config.py

```diff
@@ -0,0 +1,31 @@
+# Melissa
+from profile import data
+import importlib
+import pkgutil
+import inspect
+
+dict_tts = dict()
+for (_, name, _) in pkgutil.iter_modules(["tts_engines"]):
+    module = importlib.import_module('.' + name + ".main", "tts_engines")
+    for cls, obj in inspect.getmembers(module):
+        if inspect.isclass(obj):
```

Comment:
Here you'll need to check if it's inherited from the TTS class.

Reply:
you can check for all the previous suggestions now.